### PR TITLE
Sync Access-Control-Expose-Headers Doc

### PR DESCRIPTION
**Pull request by maoxiaoke**
sync with english version for  [Access-Control-Expose-Headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Expose-Headers) into Chinese

### CODE REVIEW

**Review by awxiaoxian2020 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/http/headers/access-control-expose-headers/index.md [hunk 1]
```diff
@@ -56,13 +72,6 @@ Access-Control-Expose-Headers: Content-Length, X-Kuma-Revision
...
 
-## 关于兼容性的注意事项
-
-- 在最新规范中提出的通配符 (\*)，尚未被如下浏览器实现：
-
-  - Chromium: [Issue 615313](https://bugs.chromium.org/p/chromium/issues/detail?id=615313)
-  - Firefox: {{bug(1309358)}}
-  - Servo: [Issue 13283](https://github.com/servo/servo/issues/13283)
 
 ## 相关内容
```
[the remark this answers is not in the document]
Comment (maoxiaoke) [reply files/zh-cn/web/http/headers/access-control-expose-headers/index.md]: 已处理

File: files/zh-cn/web/http/headers/access-control-expose-headers/index.md [hunk 2]
```diff
@@ -26,26 +26,42 @@ slug: Web/HTTP/Headers/Access-Control-Expose-Headers
...
+Access-Control-Expose-Headers: *
 ```
 
 ## 指令
 
 - \<header-name>
   - : 包含 0 个或多个除 {{Glossary("Simple response header", "simple response headers")}}（简单响应首部）之外的[首部名称](/zh-CN/docs/Web/HTTP/Headers)列表，可以暴露给外部，供页面资源使用。
 
+- \*（通配符）
+  - : 若请求没有 [HTTP Cookie](/zh-CN/docs/Web/HTTP/Cookies)或认证信息，通配符 ”*“ 会暴露所有响应头部。对于有凭证信息的请求，会被简单地识别为头部名 ”*“，没有通配的语义。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/access-control-expose-headers/index.md:39]:
```suggestion
  - : 若请求没有携带凭据（请求没有 [HTTP Cookie](/zh-CN/docs/Web/HTTP/Cookies)或认证信息），“`*`”才会被当作一个特殊的通配符。对于带有凭据的请求，会被简单地当作标头名称“`*`”，没有特殊的语义。
```

File: files/zh-cn/web/http/headers/access-control-expose-headers/index.md [hunk 3]
```diff
@@ -26,26 +26,42 @@ slug: Web/HTTP/Headers/Access-Control-Expose-Headers
...
 想要暴露一个非简单响应首部，可以这样指定：
 
 ```plain
-Access-Control-Expose-Headers: Content-Length
+Access-Control-Expose-Headers: Content-Encoding
+```
+
+想要额外暴露自定义的首部，例如 `Kuma-Revision`，可以指定多个，用逗号隔开：
+
+```plain
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/access-control-expose-headers/index.md:51]:
````suggestion
```http
````

File: files/zh-cn/web/http/headers/access-control-expose-headers/index.md [hunk 4]
```diff
@@ -26,26 +26,42 @@ slug: Web/HTTP/Headers/Access-Control-Expose-Headers
...
   - : 包含 0 个或多个除 {{Glossary("Simple response header", "simple response headers")}}（简单响应首部）之外的[首部名称](/zh-CN/docs/Web/HTTP/Headers)列表，可以暴露给外部，供页面资源使用。
 
+- \*（通配符）
+  - : 若请求没有 [HTTP Cookie](/zh-CN/docs/Web/HTTP/Cookies)或认证信息，通配符 ”*“ 会暴露所有响应头部。对于有凭证信息的请求，会被简单地识别为头部名 ”*“，没有通配的语义。
+
 ## 示例
 
 想要暴露一个非简单响应首部，可以这样指定：
 
 ```plain
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/access-control-expose-headers/index.md:45]:
```suggestion
```http
```

File: files/zh-cn/web/http/headers/access-control-expose-headers/index.md [hunk 5]
```diff
@@ -26,26 +26,42 @@ slug: Web/HTTP/Headers/Access-Control-Expose-Headers
...
+```
+
+想要额外暴露自定义的首部，例如 `Kuma-Revision`，可以指定多个，用逗号隔开：
+
+```plain
+Access-Control-Expose-Headers: Content-Encoding, Kuma-Revision
 ```
 
-想要额外暴露自定义的首部，例如 `X-Kuma-Revision`，可以指定多个，用逗号隔开：
+有凭证信息的请求，可以指定一个通配符：
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/access-control-expose-headers/index.md:55]:
```suggestion
服务器可以为不带凭据的请求响应通配符：
```

File: files/zh-cn/web/http/headers/access-control-expose-headers/index.md [hunk 6]
```diff
@@ -26,26 +26,42 @@ slug: Web/HTTP/Headers/Access-Control-Expose-Headers
...
 
 ```plain
-Access-Control-Expose-Headers: Content-Length, X-Kuma-Revision
+Access-Control-Expose-Headers: *
+```
+
+但是，“*” 对 [Authorization 头](/zh-CN/docs/Web/HTTP/Headers/Authorization) 不生效，需要显式指定：
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/access-control-expose-headers/index.md:61]: See line: https://github.com/mdn/content/blob/2344afbffd22b7932b755d6c74b8242ededdfbd2/files/en-us/web/http/headers/access-control-expose-headers/index.md?plain=1#L65

Please reflect the translation as the english content

File: files/zh-cn/web/http/headers/access-control-expose-headers/index.md [hunk 7]
```diff
@@ -26,26 +26,42 @@ slug: Web/HTTP/Headers/Access-Control-Expose-Headers
...
+
+```plain
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/access-control-expose-headers/index.md:63]: also: http

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/http/headers/access-control-expose-headers/index.md (hunk 6 above)
Comment (yin1999) [reply files/zh-cn/web/http/headers/access-control-expose-headers/index.md:61]:
```suggestion
但是，这并不会匹配 {{HTTPHeader("Authorization")}} 标头，所以如果你要暴露它，需要显式指定：
```

File: files/zh-cn/web/http/headers/access-control-expose-headers/index.md [hunk 8]
```diff
@@ -26,26 +26,42 @@ slug: Web/HTTP/Headers/Access-Control-Expose-Headers
...
 ```plain
-Access-Control-Expose-Headers: <header-name>, <header-name>, ...
+Access-Control-Expose-Headers: [<header-name>[, <header-name>]*]
+Access-Control-Expose-Headers: *
 ```
 
 ## 指令
 
 - \<header-name>
   - : 包含 0 个或多个除 {{Glossary("Simple response header", "simple response headers")}}（简单响应首部）之外的[首部名称](/zh-CN/docs/Web/HTTP/Headers)列表，可以暴露给外部，供页面资源使用。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/access-control-expose-headers/index.md:36]:
```suggestion
  - : 包含 0 个或多个除{{Glossary("Simple response header", "简单响应标头")}}之外的[标头名称](/zh-CN/docs/Web/HTTP/Headers)列表，可以暴露给外部，供页面资源使用。
```

File: files/zh-cn/web/http/headers/access-control-expose-headers/index.md (hunk 7 above)
Comment (yin1999) [reply files/zh-cn/web/http/headers/access-control-expose-headers/index.md:63]:
````suggestion
```http
````

File: files/zh-cn/web/http/headers/access-control-expose-headers/index.md [hunk 9]
```diff
@@ -26,26 +26,42 @@ slug: Web/HTTP/Headers/Access-Control-Expose-Headers
...
+想要额外暴露自定义的首部，例如 `Kuma-Revision`，可以指定多个，用逗号隔开：
+
+```plain
+Access-Control-Expose-Headers: Content-Encoding, Kuma-Revision
 ```
 
-想要额外暴露自定义的首部，例如 `X-Kuma-Revision`，可以指定多个，用逗号隔开：
+有凭证信息的请求，可以指定一个通配符：
 
 ```plain
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/access-control-expose-headers/index.md:57]:
````suggestion
```http
````

File: files/zh-cn/web/http/headers/access-control-expose-headers/index.md [hunk 10]
```diff
@@ -56,15 +72,8 @@ Access-Control-Expose-Headers: Content-Length, X-Kuma-Revision
 
 {{Compat}}
 
-## 关于兼容性的注意事项
-
-- 在最新规范中提出的通配符 (\*)，尚未被如下浏览器实现：
-
-  - Chromium: [Issue 615313](https://bugs.chromium.org/p/chromium/issues/detail?id=615313)
-  - Firefox: {{bug(1309358)}}
-  - Servo: [Issue 13283](https://github.com/servo/servo/issues/13283)
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/access-control-expose-headers/index.md:75]:
```suggestion
```

File: files/zh-cn/web/http/headers/access-control-expose-headers/index.md [hunk 11]
```diff
@@ -26,26 +26,42 @@ slug: Web/HTTP/Headers/Access-Control-Expose-Headers
...
 
 - \<header-name>
   - : 包含 0 个或多个除 {{Glossary("Simple response header", "simple response headers")}}（简单响应首部）之外的[首部名称](/zh-CN/docs/Web/HTTP/Headers)列表，可以暴露给外部，供页面资源使用。
 
+- \*（通配符）
+  - : 若请求没有 [HTTP Cookie](/zh-CN/docs/Web/HTTP/Cookies)或认证信息，通配符 ”*“ 会暴露所有响应头部。对于有凭证信息的请求，会被简单地识别为头部名 ”*“，没有通配的语义。
+
 ## 示例
 
 想要暴露一个非简单响应首部，可以这样指定：
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/access-control-expose-headers/index.md:43]:
```suggestion
想要暴露一个非简单响应标头，可以这样指定：
```

**Review by yin1999 — APPROVED** [review]
Thanks a lot. I've applied some changes to made a full async.

Welcome to the community :tada: 

We've listed some suggestions [here](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md). Looking forward to your future contribution :)

**Pull request merged**